src/ucp/core/ucp_request.h

```diff
@@ -26,6 +26,7 @@
 #include <ucp/core/ucp_worker.h>
 
 
+#define UCP_REQUEST_INVALID ((ucp_request_t*)(uintptr_t)SIZE_MAX)
```

Comment:
IMO no longer needed since we have the flag

Reply:
removed it and all occurences

Reply:
i guess we can also undo ucp_request_reset_external now

Reply:
> i guess we can also undo ucp_request_reset_external now

done